ftplugin/gsl.vim

```diff
@@ -0,0 +1,28 @@
+" Vim filetype plugin file
+" Language: Godot shading language
+" Maintainer: Maxim Kim <habamax@gmail.com>
+
+if exists("b:did_ftplugin") | finish | endif
+let b:did_ftplugin = 1
+let s:keepcpo = &cpo
+set cpo&vim
+
+let b:undo_ftplugin = '|setlocal suffixesadd<'
+      \ . '|setlocal noexpandtab<'
+
+setlocal suffixesadd=.shader
+setlocal noexpandtab
+
+if !exists("g:godot_executable")
+    if executable('godot')
+        let g:godot_executable = 'godot'
+    elseif executable('godot.exe')
+        let g:godot_executable = 'godot.exe'
+    endif
+endif
+
+command! -buffer -nargs=? -complete=customlist,godot#scene_complete GodotRun call godot#run(<q-args>)
+command! -buffer GodotRunFZF call godot#fzf_run_scene()
```

Comment:
I understand why it is here, but don't like it :)

Anyway, for now I think this is ok, I will think about better ways later.

Reply:
I also thought about it :)
Maybe move it into a function?

Reply:
In general such things must be in `plugin` folder. What if move it here?
It will not increase startup time because the plugin can be added as an optional (moved into an `opt` directory via native package manager or loaded on-demand via vim-plug)

Reply:
I don't want to pollute vim with commands that only should work for a filetype.

I think that we can safely move `if !exists("g:godot_executable")` check to autoloads `s:run_scene`:

```
    exe printf("%s%s %s",
                \ cmd,
                \ get(g:, "godot_executable", "godot"),  <-- somewhere here
                \ fnameescape(a:scene_name))
```

As for commands, not sure, let them be like this for now

Reply:
> I don't want to pollute vim with commands that only should work for a filetype.

Yes, I understand. I meant to move only `g:godot_executable` to `plugin`.
But yes, it can be moved to `s:run_scene` too. I will do it in #9 if you want to see it there.

Reply:
sure, go on!